job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContext/versionNumber.groovy

```diff
@@ -0,0 +1,15 @@
+job('example') {
+    wrappers {
+        versionNumber('1.0.0.${BUILDS_ALL_TIME}', 'VERSION') {
+            startDate()
```

Comment:
try to find a better example with some values instead of just calling all methods without options